XIPs/xip-21-transaction-reference-content-type.md

```diff
@@ -35,17 +35,13 @@ The goal of the transaction reference content type is to provide transaction det
 ```ts
 type TransactionReference = {
   /**
-   * The namespace for the networkId
+   * The CAIP-2 chain ID where the transaction happened in the format `<namespace>:<reference>`
    */
-  namespace?: string;
-  /**
-   * The networkId for the transaction, in decimal or hexidecimal format
-   */
-  networkId: number | string;
+  chainId?: string;
```

Comment:
Should we make the chainId required?
If not, the behaviour must be documented, for instance, if undefined it defaults to `eip155:1` (Ethereum mainnet).
I'd personnally prefer it to be required so that there is (slightly) less case to manage for builder implementing the specification and, in a chain-agnostic context, there is no preference to a give chain (the default one if undefined)

Reply:
Dear @aurelticot - thank you for your initiative in keeping an eye on this PR! I'd like to suggest that when the original author of this XIP returns in late April/early May - we set up a time for you and Ry to sync up just to cross t's and dot i's, as they say.

Might you be open to this? I will take the action item to bring us together at that time! <3

Reply:
Yep sure. Thanks

Reply:
Captured here for resolution upon Ry's return: https://community.xmtp.org/t/xip-21-on-chain-transaction-reference-content-type/532/11?u=jha